query-sql/src/lib.rs

```diff
@@ -152,6 +152,14 @@ pub enum TableOrSubquery {
     Table(SourceAlias),
     Union(Vec<SelectQuery>, TableAlias),
     Subquery(Box<SelectQuery>),
+    Values(Values, TableAlias),
+}
+
+pub enum Values {
+    /// Like "VALUES (0, 1), (2, 3), ...".
+    Unnamed(Vec<Vec<TypedValue>>),
+    /// Like "SELECT 0 AS x, SELECT 0 AS y WHERE 0 UNION ALL VALUES (0, 1), (2, 3), ...".
+    Named(Vec<Variable>, Vec<Vec<TypedValue>>),
```

Comment:
It also occurs to me that the preferred way of doing this kind of thing on modern computers is not to have multiple vecs, but to have a single vec and a dimension argument. Better locality.